Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7011,6 +7043,2029 @@ def panorama_install_file_content_update_command(args: dict):
+
+    def _setup(self):
+        # xpaths
+        self._xpaths.add_profile(value="/rules")
+
+        # params
+        params = []
+        params.append(
+            VersionedParamPath("severity", vartype="member", path="severity")
+        )
+        params.append(
+            VersionedParamPath("is_reset_both", vartype="exist", path="action/reset-both")
+        )
+        params.append(
+            VersionedParamPath("is_reset_client", vartype="exist", path="action/reset-client")
+        )
+        params.append(
+            VersionedParamPath("is_reset_server", vartype="exist", path="action/reset-server")
+        )
+        params.append(
+            VersionedParamPath("is_alert", vartype="exist", path="action/alert")
+        )
+        params.append(
+            VersionedParamPath("is_default", vartype="exist", path="action/default")
+        )
+        params.append(
+            VersionedParamPath("is_allow", vartype="exist", path="action/allow")
+        )
+        params.append(
+            VersionedParamPath("is_drop", vartype="exist", path="action/drop")
+        )
+        params.append(
+            VersionedParamPath("is_block_ip", vartype="exist", path="action/block-ip")
+        )
+        self._params = tuple(params)
+
+
+class VulnerabilityProfile(CustomVersionedPanObject):
+    """Vulnerability Profile Group Object
+
+    Args:
+        name (str): Name of the object
+    """
+
+    ROOT = Root.VSYS  # pylint: disable=E1101
+    SUFFIX = ENTRY
+    CHILDTYPES = (VulnerabilityProfileRule,)
+
+    def _setup(self):
+        # xpaths
+        self._xpaths.add_profile(value="/profiles/vulnerability")
+
+        # params
+        params = []
+
+        self._params = tuple(params)
+
+
+class URLFilteringProfile(VersionedPanObject):
+    """URL Filtering profile
+
+    :param block: Block URL categories
+    :param alert: Alert URL categories
+    :param credential_enforce_block: Categories blocking credentials
+    :param credential_enforce_alert: Categories alerting on credentials
+    """
+
+    ROOT = Root.VSYS  # pylint: disable=E1101
+    SUFFIX = ENTRY
+
+    def _setup(self):
+        # xpaths
+        self._xpaths.add_profile(value="/profiles/url-filtering")
+
+        # params
+        params = []
+        params.append(
+            VersionedParamPath("block", vartype="member", path="block")
+        )
+
+        params.append(
+            VersionedParamPath("alert", vartype="member", path="alert")
+        )
+
+        params.append(
+            VersionedParamPath("credential_enforce_alert", vartype="member",
+                               path="credential-enforcement/alert")
+        )
+        params.append(
+            VersionedParamPath("credential_enforce_block", vartype="member",
+                               path="credential-enforcement/block")
+        )
+        self._params = tuple(params)
+
+
+def run_op_command(device: Union[Panorama, Firewall], cmd: str, **kwargs) -> Element:
```

Comment:
what is an ```op``` command?

Reply:
Op = operational, the PANOS API refers to it as an "OP" command